src/content/en/2020/resource-hints.md

```diff
@@ -8,16 +8,336 @@ authors: [Zizzamia]
+
+## Service Workers
+
+For both `preload` and `prefetch` we&#39;ve had an increase in adoption when the page is controlled by a [Service Worker](https://developers.google.com/web/fundamentals/primers/service-workers). This is because of the potential to both improve the resource prioritization by preloading when the Service Worker is not active yet and intelligently prefetching future resources while letting the Service Worker cache them before they&#39;re needed by the user.
+
+// TODO @zizzamia Add chart 4
+
+For `preload` on desktop we have an outstanding 47% rate of adoption and `prefetch` a 10% rate of adoption. In both cases the data is much higher compared to average adoption without a Service Worker.
+
+As mentioned earlier, the [PRPL Pattern](https://addyosmani.com/blog/the-prpl-pattern/) will play a significant role in the coming years in how we combine Resource Hints with the Service Worker caching strategy.
+
+## Future
+
+Let&#39;s dive into a couple of experimental hints. Very close to release we have Priority Hints, at the moment actively experimented with the web community. Lastly we have the 103 Early Hints in HTTP/2, which is still in early inception and there are a few players like Chrome and Fastly collaborating for upcoming test trials.
+
+### Priority Hints
+
+[Priority hints](https://developers.google.com/web/updates/2019/02/priority-hints) are an API for expressing the fetch priority of a resource: high, low, or auto. They can be used to help deprioritize images (e.g. inside a Carousel), re-prioritize scripts and even help de-prioritize fetches.
+
+This new hint can be used either as an HTML tag or by changing the priority of fetch requests via the `importance` option, which takes the same values as the HTML attribute.
+
+With `preload` and `prefetch`, the priority is set by the browser depending on the type of resource. By using Priority Hints we can force the browser to change the default option.
+
+{{ figure_markup(
+  caption="The rate of priority hint adoption on mobile",
+  content="0.77%",
+  classes="big-number",
+  sheets_gid="1805612941",
+  sql_file="// TODO @zizzamia Add sql"
+) }}
+
+So far only 0.77% websites adopted this new hint as Chrome is still [actively](https://www.chromestatus.com/features/5273474901737472) experimenting, and at the time of this article&#39;s release the feature is on-hold.
+
+{{ figure_markup(
+  caption="Priority hints on mobile are on script elements",
+  content="80%",
+  classes="big-number",
+  sheets_gid="1805612941",
+  sql_file="// TODO @zizzamia Add sql"
+) }}
+
+The largest use is with script elements, which is unsurprising as the number of JS primary and third-party files continues to grow.
+
+{{ figure_markup(
+  caption="Priority hints on mobile have &quot;low&quot; importance",
```

Comment:
```suggestion
  caption="Priority hints on mobile have \"low\" importance",
```

Reply:
Nice, great catch! I just added the fix requested.